docs/docker.md

```diff
@@ -20,11 +20,11 @@ The following steps can be used to setup Stringer on Docker, using a Postgres da
 docker run --detach \
     --name stringer-postgres \
     --restart always \
-    --volume /srv/stringer/data:/var/lib/postgresql/data \
+    --volume ~/stringer:/var/lib/postgresql/data \
     --net stringer \
     -e POSTGRES_PASSWORD=myPassword \
     -e POSTGRES_DB=stringer \
-    postgres:9.5-alpine
+    postgres:12-alpine
```

Comment:
We discussed briefly on your other PR, but do you have any sense of what steps a user will need to take in order to preserve the data in their Postgres instance while upgrading?

Reply:
The most straightforward way to upgrade would be:
- `docker -it -u postgres stringer-postgres pg_dumpall > pg_dump.bak`
- `mv /srv/stringer/data /srv/stringer/data_backup`
- `docker compose up -d` (this will rebuild the postgres container with the new version of postgres, and will start from scratch because there's no `/srv/stringer/data` directory anymore)
- `docker cp pgdump.bak stringer-postgres:/pgdump.bak`
- `docker exec -it stringer-postgres chown postgres:postgres /pgdump.bak`
- `docker exec -u postgres -it stringer-postgres psql -f /pgdump.bak postgres`

Essentially, dumping the database to a file + creating a backup, recreating the database docker container, then restoring the database from file. [According to the postgres docs](https://www.postgresql.org/docs/current/upgrading.html), this is "The traditional method for moving data, [...] though this can be slow." In our use case, I don't think this matters that much; stringer doesn't have enough data stored in the db to really make it a slow operation. The reasoning for following this upgrade path is because major versions of postgres can contain changes to the underlying data structure on disk, and so we cannot copy the data directory over directly, since the structure may have (and, going from 9.5 to 16, most probably has) changed.

An alternate upgrade path, also described in the postgres docs, is using `pg_upgrade`. I haven't used this personally, but this seems to require a lot more work with installing multiple binaries for postgres in a single host/container. I think that if an end user knows about this upgrade path, they most probably know how to use it, or can figure it out for themselves, so I don't think we need to concern ourselves with this.

What do you think? We can add the above commands to docs, or make this into its own release with upgrade instructions?

Reply:
@guidopetri that sounds reasonable to me. Is there likely to be any issue with exporting and reimporting data across so many versions?

Reply:
No; from the docs linked above as well: 

> Current releases of the dump programs can read data from any server version back to 9.2.